src/devices/schneider_electric.ts

```diff
@@ -108,6 +120,176 @@ function switchActions(endpoint?: string) {
     });
 }
 
+const getVisaConfigurationCluster = (
+    enumDataType: Zcl.DataType.ENUM8 | Zcl.DataType.UINT8,
+) => ({
+    ID: 0xfc04,
+    manufacturerCode: Zcl.ManufacturerCode.SCHNEIDER_ELECTRIC,
+    attributes: {
+        indicatorLuminanceLevel: {ID: 0x0000, type: enumDataType},
+        indicatorColor: {ID: 0x0001, type: enumDataType},
+        indicatorMode: {ID: 0x0002, type: enumDataType},
+        motorTypeChannel1: {ID: 0x0003, type: Zcl.DataType.UINT8},
+        motorTypeChannel2: {ID: 0x0004, type: Zcl.DataType.UINT8},
+        curtainStatusChannel1: {ID: 0x0005, type: Zcl.DataType.UINT8},
+        curtainStatusChannel2: {ID: 0x0006, type: Zcl.DataType.UINT8},
+    },
+    commands: {},
+    commandsResponse: {},
+});
+
+function visaConfigIndicatorLuminanceLevel(endpoint?: string): ModernExtend {
```

Comment:
Could you structure all the modernExtend similar to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/sonoff.ts#L36?